src/components/step/StepLayout/StepLayout.css

```diff
@@ -0,0 +1,37 @@
+.step-container {
+    background-color: #FFFFFF;
+}
+
+.step-container div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,form,fieldset,input,textarea,p,blockquote,th,td { 
```

Comment:
@ashleyrtate I think this rule is wrong and these are supposed to all be embedded under .step-container right?  So needs to be .step-container div, .step-container dl, .step-container dt, etc.

Reply:
👍

Reply:
What issue did you see from this? Is it screwing up a view style?